service/matching/matchingEngine_test.go

```diff
@@ -330,12 +330,18 @@ func (s *matchingEngineSuite) PollForTasksEmptyResultTest(taskType int) {
 			DescRequest: &workflow.DescribeTaskListRequest{
 				TaskList:     taskList,
 				TaskListType: &taskListType,
+				IsAdmin:      common.BoolPtr(true),
 			},
 		})
 		s.NoError(err)
 		s.Equal(1, len(descResp.Pollers))
 		s.Equal(identity, descResp.Pollers[0].GetIdentity())
 		s.NotEmpty(descResp.Pollers[0].GetLastAccessTime())
+		s.Zero(descResp.GetBacklogCountHint())
```

Comment:
There are two more tests I can see as useful here
1. A test where IsAdmin = false and you verify that the expected fields are not set (you actually want them to be null not just defaults)
2. A test where you get a response back that contains some non-default values. Right now this test will pass even if the response is not set because GetBacklogCountHint will return the default value of zero if backlogHint is not set.